core/station/impl/src/services/system.rs

```diff
@@ -662,177 +1013,95 @@ mod install_canister_handlers {
 
         // adds the default request policies which sets safe defaults for the canister
         for policy in policies_to_create.iter() {
-            REQUEST_POLICY_SERVICE
-                .add_request_policy(AddRequestPolicyOperationInput {
-                    specifier: policy.0.to_owned(),
-                    rule: policy.1.to_owned(),
-                })
-                .map_err(|e| format!("Failed to add default request policy: {:?}", e))?;
+            REQUEST_POLICY_SERVICE.add_request_policy(AddRequestPolicyOperationInput {
+                specifier: policy.0.to_owned(),
+                rule: policy.1.to_owned(),
+            })?;
         }
 
         // adds the default permissions which sets safe defaults for the canister
         for policy in DEFAULT_PERMISSIONS.iter() {
             let allow = policy.0.to_owned();
-            PERMISSION_SERVICE
-                .edit_permission(EditPermissionOperationInput {
-                    auth_scope: Some(allow.auth_scope),
-                    user_groups: Some(allow.user_groups),
-                    users: Some(allow.users),
-                    resource: policy.1.to_owned(),
-                })
-                .map_err(|e| format!("Failed to add default permission: {:?}", e))?;
+            PERMISSION_SERVICE.edit_permission(EditPermissionOperationInput {
+                auth_scope: Some(allow.auth_scope),
+                user_groups: Some(allow.user_groups),
+                users: Some(allow.users),
+                resource: policy.1.to_owned(),
+            })?;
         }
 
         Ok(())
     }
 
+    #[allow(unused)]
     // Registers the initial accounts of the canister during the canister initialization.
+    // Used
```

Comment:
this looks like an unfinished comment

Reply:
Looks resolved to me: https://github.com/dfinity/orbit/blob/kepler/pen-411-configurable-station-init/core/station/impl/src/services/system.rs#L1039

Reply:
I'm sorry, I missed it when looking through the last two commits.